RGB LED brightness control V1.0 Design/code/APP/app.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * 	File:					app.c
+ *	Module:					APP
+ *	Description: 		source file 
+ * 	Author: 			HAZEM ASHRAF
+ */ 
+
+/*=========FILE INCLUSION=========*/
+#include "app.h"
+
+/*=========MACRO DEFINITIONS=========*/
+#define MAX_PRESS 		4
+#define RESET 			0 
+#define PERODIC_TIME 	500
+#define _90_PERCENT		90
+#define _60_PERCENT		60
+#define _30_PERCENT		30
+#define STOP			0
+#define DEBOUNCE_TIME 	50
+#define BUTTON_PIN		ENU_PIN_0
+#define GREEN_LED_PIN	ENU_PIN_3
+
+/*=========EXTERN VARIABLES=========*/
+
+/*=========GLOBAL VARIABLES=========*/
+
+/*=========TYPE DEFINITIONS=========*/
+
+/*=========APIs DEFINITIONS=========*/
+void app_init(void)
+{
+	LED_init(ENU_PORT_F, GREEN_LED_PIN);
+	BUTTON_init(ENU_PORT_F, BUTTON_PIN);
+	PWM_Init(ENU_PORT_F,GREEN_LED_PIN,TIMER0_ID);
+}
+
+void app_start(void)
+{
+	boolean state_flag 	= FALSE;
+	uint8 button_value	= 0;
+	boolean flag		= TRUE;
+	uint8 press_count	= RESET;
+	uint8 u8_duty_cycle	= 0;
+
+  while(1)
+	{
+		
+		BUTTON_digitalRead(ENU_PORT_F,BUTTON_PIN,&button_value);
+		if(button_value == LOGIC_HIGH)
+		{
+			delay_ms(TIMER1_ID,DEBOUNCE_TIME);
+			BUTTON_digitalRead(ENU_PORT_F,BUTTON_PIN,&button_value);
+			if(button_value == LOGIC_HIGH)
+			{
+				if(flag == FALSE)
+				{
+					state_flag =TRUE;
+					press_count++;
+					if(press_count == 1)
```

Comment:
You shouldn't use magic numbers

Reply:
DONE, you can check again after modifications.

Reply:
I re-reviewed your code and saw the requested changes and would like to thank you for your efforts.

Please note that:
-The changes you made after our review will not increase your score as we told you our comments to not do it again and lose marks in the next deliveries.

Reply:
thank you for your support.